Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM node:12-alpine AS build
+# packages needed for node-gyp and dupe-krill
+RUN apk add --update --no-cache python make g++ rust cargo
+# use a non-root user to make commands such as npm post-install scripts safer
+USER node
+ENV NPM_CONFIG_PREFIX=/home/node/.npm-global
+ENV PATH=$PATH:/home/node/.npm-global/bin:/home/node/.cargo/bin
+# RUN cargo install dupe-krill
+# copy by default will make the root user own the files, we want the node user to own the files
+COPY --chown=node:node package*.json .snyk /home/node/app/
```

Comment:
I need to understand users/groups better

Reply:
When I run this, `/home/node/` exists, but `/home/node/app/` doesn't at this point.

Reply:
You'll need to copy `.eleventy.js` as well.

Reply:
That happens on like 13 ` COPY --chown=node:node . /home/node/app/ `